project.godot

```diff
@@ -442,6 +442,10 @@ file_logging/max_log_files=8
 
 limits/message_queue/max_size_kb=8192
 
+[mono]
+
+project/assembly_name="Thrive"
```

Comment:
Is there a reason this change is here?

Reply:
My Godot always wants to add it. And hh seems to have accepted it and merged these lines into master in my autoevo II pr.